baselines/dom.generated.d.ts

```diff
@@ -4761,16 +4761,16 @@ interface Document extends Node, DocumentAndElementEventHandlers, DocumentOrShad
      * Returns a reference to the first object with the specified value of the ID attribute.
      * @param elementId String that specifies the ID value.
      */
-    getElementById(elementId: string): HTMLElement | null;
+    getElementById<E extends Element = HTMLElement>(elementId: string): E | null;
     /**
      * Returns a HTMLCollection of the elements in the object on which the method was invoked (a document or an element) that have all the classes given by classNames. The classNames argument is interpreted as a space-separated list of classes.
      */
-    getElementsByClassName(classNames: string): HTMLCollectionOf<Element>;
+    getElementsByClassName<E extends Element = HTMLElement>(classNames: string): HTMLCollectionOf<E>;
     /**
      * Gets a collection of objects based on the value of the NAME or ID attribute.
      * @param elementName Gets a collection of objects based on the value of the NAME or ID attribute.
      */
-    getElementsByName(elementName: string): NodeListOf<HTMLElement>;
+    getElementsByName<E extends Element = Element>(elementName: string): NodeListOf<E>;
```

Comment:
We think this should should go back `HTMLElement` - https://github.com/microsoft/TypeScript/issues/7709 ( of all of the `getElementsX` this one is the least controversial as it can only be HTMLElements anyway)

Other than that, then we're good to merge 👍🏻

Reply:
@orta Changes made! Ready for approval.